orca-keel/src/main/kotlin/com/netflix/spinnaker/orca/keel/task/ImportDeliveryConfigTask.kt

```diff
@@ -156,13 +169,18 @@ constructor(
     var manifest: String? = "spinnaker.yml",
     var ref: String? = null,
     var attempt: Int = 1,
-    val maxRetries: Int? = MAX_RETRIES
+    val maxRetries: Int? = MAX_RETRIES,
+    var errorFromLastAttempt: String? = null
   )
 
   fun ImportDeliveryConfigContext.incrementAttempt() = this.also { attempt += 1 }
   fun ImportDeliveryConfigContext.toMap() = objectMapper.convertValue<Map<String, Any?>>(this)
 
   companion object {
     const val MAX_RETRIES = 5
+    const val UNAUTHORIZED_SCM_ACCESS_MESSAGE =
+      "HTTP 401 response received while trying to read your delivery config file. " +
+      "Spinnaker may be missing permissions in your source code repository to read the file. " +
+      "Please consult your Spinnaker operator for details on how to add those permissions."
```

Comment:
This last line is probably not necessary - I think it just adds confusion.

Reply:
OK, will remove.